src/widget/wtracktableview.cpp

```diff
@@ -508,9 +508,20 @@ void WTrackTableView::slotShowHideTrackMenu(bool show) {
         return;
     }
     if (show) {
+        // Show at current index if it's valid, else at cursor position
+        QPoint evPos;
+        auto currIdx = currentIndex();
+        if (currIdx.isValid()) {
+            evPos = mapToGlobal(visualRect(currIdx).center());
+            // The viewport start below the header, but mapToGlobal() apparently
+            // uses the rect() as reference. Add header height to y and we're good.
+            evPos += QPoint(0, horizontalHeader()->height());
+        } else {
+            evPos = QCursor::pos();
```

Comment:
might as well use `getSelectedRows().first()` here (if it's not empty!)

Reply:
naah, use first selected index (that's what contextMenuEvent() uses to load the track menu), then shift to column of current index if that is valid.